README.md

```diff
@@ -1,3 +1,67 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/distroless.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/distroless)
 
 # This repository contains source code used to build "distroless" docker images.
+
+"Distroless" images contain only your application and it's runtime dependencies.
+They do not contain package managers, shells any other programs you would expect to find in a standard Linux distribution.
+
+For more information, see this talk.
+
+## Why should I use distroless images?
+
+## How do I use distroless images?
+
+These images are built using the [bazel](https://bazel.build) tool, but they can also be used through other Docker image build tooling.
+
+### Docker
+
+Docker multi-stage builds make using distroless images easy.
+Follow these steps to get started:
+
+* Pick the right base image for your application stack
+  We publish the following distroless base images on `gcr.io`:
+  <add links to docs>
+
+    * [gcr.io/distroless/base]()
```

Comment:
I think we should probably have a README under each directory that talks about its expected usage (e.g. with java using the jar via CMD, with jetty where to put the webapp).